modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/coordination/SlotAgent.java

```diff
@@ -36,19 +35,19 @@
 	 * @param storageQueueName name of storage queue
 	 * @param assignedNodeId Assigned node id of slot
 	 */
-	public void createSlot(long startMessageId, long endMessageId,
+	void createSlot(long startMessageId, long endMessageId,
```

Comment:
It is permitted, but discouraged as a matter of style, to redundantly specify the public and/or abstract modifier for a method declared in an interface.

http://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4